packages/adblocker/src/filters/network.ts

```diff
@@ -888,6 +889,16 @@ export default class NetworkFilter implements IFilter {
             mask = setBit(mask, NETWORK_FILTER_MASK.isReplace);
             optionValue = value;
 
+            break;
+          case 'removeparam':
```

Comment:
Nit. Since it would be invalid to have filters with a combination of `removeparam` and another `optionValue`, can we add a check to reject filters that try to combine at least two of these options?

Reply:
That's good idea, I think we can do it in a separate PR instead. `removeparam` is not only the filter option involving `optionValue`.

Reply:
Sounds good, let's please open an issue to not forget about it.

Reply:
Opened here https://github.com/ghostery/adblocker/issues/4763